### PR TITLE
MGDCTRS-939: chore: update as per UX suggestions

### PR DESCRIPTION
**Pull request by indraraj**
Screenshots 
image

image

image

### CODE REVIEW

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx [hunk 1]
```diff
@@ -229,6 +231,25 @@ export const ConnectorDrawerPanelContent: FunctionComponent<ConnectorDrawerPanel
...
+              <Trans i18nKey={'supportEmailMsg'}>
+                You can still get help by emailing us at
+                <ClipboardCopy
+                  hoverTip="Copy"
+                  clickTip="Copied"
+                  variant="inline-compact"
+                >
+                  rhosak-eval-support@redhat.com
+                </ClipboardCopy>
+                . This mailing list is monitored by the Red Hat OpenShift
```
Comment (uidoyen) [inline src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:247]: @indraraj Can we use these texts with i18n?

File: src/app/components/ConnectorsTable/ConnectorsTable.tsx [hunk 2]
```diff
@@ -132,7 +141,44 @@ export const ConnectorsTableRow: FunctionComponent<ConnectorsTableRowProps> = ({
...
       {/* <Td dataLabel={t('Category')}>{category}</Td> */}
       <Td dataLabel={t('status')}>
-        <ConnectorStatus name={name} status={status} />
+        <Popover
+          aria-label="Failed connector popover"
+          position={PopoverPosition.auto}
+          hideOnOutsideClick={true}
+          headerContent={
+            <h1 className="connectors-failed_pop_over">
+              <ExclamationCircleIcon /> Failed
```
Comment (uidoyen) [inline src/app/components/ConnectorsTable/ConnectorsTable.tsx:150]: here too.

**Review by uidoyen — CHANGES_REQUESTED** [review]
The rest looks great!

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx (hunk 1 above)
Comment (indraraj) [reply src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:247]: I am using here the `<Trans i18nKey={'supportEmailMsg'}>` to use the i18n, let me know if it's incorrect.

File: src/app/components/ConnectorsTable/ConnectorsTable.tsx (hunk 2 above)
Comment (indraraj) [reply src/app/components/ConnectorsTable/ConnectorsTable.tsx:150]: sure, will update.

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx (hunk 1 above)
Comment (uidoyen) [reply src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:247]: Yeah, my bad.

**Review by uidoyen — APPROVED** [review]

**Pull request merged**